src/main/java/com/jvm_bloggers/frontend/admin_area/AdminSocialChannelsPage.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:wicket="http://wicket.apache.org/dtds.data/wicket-xhtml1.5-strict.dtd"
+>
+<head>
+</head>
+<body>
+
+<wicket:extend>
+    <div id="page-wrapper">
+    <!-- /.row -->
+    <div class="row">
+        <div class="col-lg-12">
+            <div class="panel panel-primary">
+                <div class="panel-heading">
+                    <i class="fa fa-facebook fa-fw"></i> Post on Facebook
+                </div>
+                <!-- /.panel-heading -->
+                <div class="panel-body">
+        <form action="" wicket:id="facebookPostForm">
```

Comment:
Generally, this form needs some more love to look nicer. Please look at `http://localhost:8080/mailing-address` in Admin Panel or generally at https://blackrockdigital.github.io/startbootstrap-sb-admin-2/pages/forms.html (template we are using).